payment/tests.py

```diff
@@ -0,0 +1,107 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
+
+from payment.models import Payment
+from payment.serializers import PaymentSerializer
+
+
+class PaymentModelTestCase(TestCase):
+    def setUp(self):
+        Payment.objects.create(
+            status="PAID",
+            type="PAYMENT",
+            borrowing_id=1,
+            session_url="https://example.com",
+            session_id="test_session_id",
+            money_to_pay=100.00,
+        )
+
+    def test_payment_str_representation(self):
+        payment = Payment.objects.get(borrowing_id=1)
+        self.assertEqual(str(payment), "Paid - Payment - Borrowing ID: 1")
+
+
+class PaymentSerializerTestCase(TestCase):
+    def setUp(self):
+        self.payment_data = {
+            "status": "PENDING",
+            "type": "FINE",
+            "borrowing_id": 2,
+            "session_url": "https://test.com",
+            "session_id": "test_session_id_2",
+        }
+        self.serializer = PaymentSerializer(data=self.payment_data)
+
+    def test_serializer_not_valid(self):
+        self.assertFalse(self.serializer.is_valid())
+
+    def test_serializer_save(self):
+        self.payment_data["money_to_pay"] = 50
+        self.assertTrue(self.serializer.is_valid())
+        self.serializer.save()
+        payment = Payment.objects.get(borrowing_id=2)
+        self.assertEqual(payment.status, "PENDING")
+
+
+class PaymentViewSetTestCase(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username="testuser", password="12345")
+        self.token = Token.objects.create(user=self.user)
+        self.client.credentials(HTTP_AUTHORIZATION=f"Token {self.token.key}")
```

Comment:
You can use self.client.force_authenticate and pass the user instance there instead of token authorization. And after that, you won't have to add "rest_framework.authtoken" in the settings. We need to stick to JWT authentication.